src/export/Package.ts

```diff
@@ -35,9 +35,29 @@ export class Package implements Fishable {
       switch (type) {
         case Type.Profile:
           def = this.profiles.find(p => p.id === item || p.name === item || p.url === item);
+          if (!def) {
+            def = this.instances.find(
+              i =>
+                (i.id === item || i._instanceMeta.name === item || i.url === item) &&
+                i._instanceMeta.usage === 'Definition' &&
+                i.resourceType === 'StructureDefinition' &&
+                i.derivation === 'constraint' &&
+                i.type !== 'Extension'
+            );
+          }
           break;
         case Type.Extension:
           def = this.extensions.find(e => e.id === item || e.name === item || e.url === item);
+          if (!def) {
+            def = this.instances.find(
+              i =>
+                (i.id === item || i._instanceMeta.name === item || i.url === item) &&
+                i._instanceMeta.usage === 'Definition' &&
+                i.resourceType === 'StructureDefinition' &&
+                i.derivation === 'constraint' &&
+                i.type === 'Extension'
```

Comment:
Similar comment as above.  It might be more efficient to check `i.derivation === 'constraint' && i.type === 'Extension'` first, since this seems it will eliminate 99% of cases right away.